google-cloud-clients/google-cloud-core/src/main/java/com/google/cloud/Date.java

```diff
@@ -68,7 +70,39 @@ public static Date parseDate(String date) {
     int dayOfMonth = Integer.parseInt(matcher.group(3));
     return new Date(year, month, dayOfMonth);
   }
+  
+  /**
+  * @param date
+  * @return LocalDate
+  */
+  public static LocalDate toDate(Date date){
```

Comment:
`Date::toDate` feels odd. I would suggest `toLocalDate`.

Reply:
This is a one-liner. Does it carry its weight? It can be inlined almost trivially.